build.sbt

```diff
@@ -226,6 +226,7 @@ lazy val docs = http4sProject("docs")
   .settings(tutSettings)
   .settings(
     libraryDependencies <+= scalazVersion {szv => argonautShapeless(szv) },
+    libraryDependencies += "org.reactormonk" %% "cryptobits" % "1.0",
```

Comment:
We keep the other dependencies in Http4sBuild, so if more than one project needs them, we get the same versions across the board.  Might wish to do that for consistency.